lib/config.js

```diff
@@ -86,6 +86,10 @@ const getConfig = (config) => {
     config.password = process.env.PASSWORD
   }
 
+  if (process.env.KIT_TEST_DIR) {
+    config.isAcceptanceTest = true
+  }
+
```

Comment:
This isn't used anywhere, can we remove it?

Reply:
Done